transport_nantes/gom_jabbar/models.py

```diff
@@ -60,3 +63,19 @@ class Visit(models.Model):
     user_is_authenticated = models.BooleanField(blank=True, default=False)
 
     timestamp = models.DateTimeField(auto_now=True)
+
+
+class Visitor(models.Model):
+    """Track visitors to the site.
+
+    A visitor is a set of statisticcs concerning a set of visits with
```

Comment:
statisticcs -> statistics

Reply:
Also, a single visitor instance has 2 IntegersFields.
I don't understand how this works ? Did you mean to increase that counter ? Like if one person visits the website and is human, you can tell they visited the website X times, X being Visitor.visits_from_humans ?

Please give me an example of how you would like to use this model, I'm having troubles seeing the use cases here